src/routes/resource.js

```diff
@@ -4,7 +4,12 @@ import response   from '../response';
 
 import * as ERR   from '../errors';
 
-module.exports = function resource(endpoint, exclude, associations = []) {
+module.exports = function resource(endpoint, exclude) {
+
+  let associations = [];
+  db().then(models => {
+    associations = Object.keys(models[endpoint].associations);
```

Comment:
It's quite unlikely, but this can race with the router handlers.

Instead of settings associations in a global, I would create a function like:

```javascript
const assocciations = model => return Object.keys(models[endpoint].associations);
```

And then call it inside the router handlers.